fiftyone/core/collections.py

```diff
@@ -5540,6 +5540,77 @@ def export(
         if archive_path is not None:
             etau.make_archive(export_dir, archive_path, cleanup=True)
 
+    def annotate(
+        self,
+        backend="cvat",
+        label_schema=None,
+        label_field=None,
+        label_type=None,
+        classes=None,
+        attributes=True,
+        media_field="filepath",
+        launch_editor=False,
+        **kwargs,
+    ):
+        """Exports the samples and a label field to the given annotation
+        backend.
+
```

Comment:
Perhaps add an `Examples::` section

Reply:
I added a link to the integration docs page to cover the examples request here